tests/api2/test_alert.py

```diff
@@ -0,0 +1,82 @@
+from time import sleep
+
+import pytest
+from pytest_dependency import depends
+
+from auto_config import pool_name
+from middlewared.test.integration.utils import call, ssh
+
+
+ID_PATH = "/dev/disk/by-partuuid/"
+
+
+def get_alert_by_id(alert_id):
+    return next(filter(lambda alert: alert["id"] == alert_id, call("alert.list")), None)
+
+
+@pytest.mark.dependency(name="degrade_pool")
```

Comment:
Let's not use dependencies where we can use fixtures

Reply:
Just to be clear, do you want me to turn both tests that are marked as dependencies into fixtures and then call for those fixtures in the other tests? In the case that the fixtures fail, instead of skipping the tests that depend on them they would all fail. The fixtures would also be run every time they're called for in a test.